firebase.json

```diff
@@ -128,7 +128,7 @@
       { "source": "/flutter", "destination": "https://flutter.io", "type": 301 },
 
       { "source": "/go/experiments", "destination": "/tools/experiment-flags", "type": 301 },
-      { "source": "/go/null-safety-migration", "destination": "https://github.com/dart-lang/sdk/tree/master/pkg/nnbd_migration", "type": 301 },
+      { "source": "/go/null-safety-migration", "destination": "https://kw-www-dartlang-1.firebaseapp.com/null-safety/migration-guide", "type": 302 },
```

Comment:
I'm curious if there was a reason to change this from a 301 to a 302 (the other redirects in this file use a 301).

Reply:
302 redirects are supposed to be temporary, so I used that because we'll change the target soon. It's my (perhaps misguided) hope that search engines might be more likely to pick up a change to a 302 redirect.